wire/modules/AdminTheme/AdminThemeUikit/AdminThemeUikit.module

```diff
@@ -578,6 +578,24 @@ class AdminThemeUikit extends AdminThemeFramework implements Module, Configurabl
 	 *
 	 */
 
+	/**
+	 * Render file
+	 * 
+	 * You can either place custom files in /site/templates/AdminThemeUikit or you
+	 * can specify a custom folder via $config->AdminThemeUikitTemplates
+	 * 
+	 * Usage:
+	 * $adminTheme->render('_head.php');
+	 * @return string
+	 */
+	public function ___render($file) {
+		$config = $this->wire->config;
+		$path = $config->paths->templates."AdminThemeUikit/$file";
+		if(!is_file($path)) $path = rtrim($config->AdminThemeUikitTemplates,"/")."/$file";
```

Comment:
Note: this is likely to cause deprecation errors in PHP 8.1, since by default non-defined config properties are `null` and "Passing null to non-nullable internal function parameters is deprecated."

Reply:
Thx for reviewing my code teppo :) What do you mean? I don't understand. Where am I passing null ?

Reply:
Thx, so we should do a
```
($config->AdminThemeUikitTemplates ?: '')
```
?

Reply:
That should do it :)

Reply:
Did you see the new commit? https://github.com/processwire/processwire/pull/215/commits/01ad77b18c3a468da7f7b4b6a953666dd8539e50

Reply:
Nope, but that should work as well :P

Reply:
Just had a quick look and it looks a bit weird — did you forget a closing brace `}` or am I missing something? Does that code work?